src/scrabble.js

```diff
@@ -1,5 +1,137 @@
-function scrabble() {
+function isAlpha(str) {
+  return (
+    /^[A-Za-z]*$/.test(str) ||
+    str === '{' ||
+    str === '}' ||
+    str === '[' ||
+    str === ']'
+  )
+}
+
+function getLettersInBracket(fromIndex, toIndex, scrabbleArray) {
+  return scrabbleArray.slice(fromIndex, toIndex)
+}
 
+function getTotalScore(k, scrabbleArray) {
```

Comment:
Was there a reason for using the parameter of k for your getTotalScore function? It can get a bit confusing later on as to whether it is the letter 'k' or the parameter k that is being passed in

Reply:
Yeah, that was probably something i should change as it made sense in my last iteration